spring-cloud-dataflow-admin/src/main/java/org/springframework/cloud/dataflow/admin/config/ArtifactRegistryPopulator.java

```diff
@@ -115,8 +125,8 @@ private void populateDefault(String name, ArtifactType type) {
 	 * @return default coordinates for the provided module
 	 */
 	private ArtifactCoordinates defaultTaskCoordinatesFor(String moduleName) {
-		return ArtifactCoordinates.parse(String.format("%s:%s:%s",
-				DEFAULT_TASK_GROUP_ID, moduleName, DEFAULT_VERSION));
+		return ArtifactCoordinates.parse(String.format("%s:%s:%s:%s:%s",
+				DEFAULT_TASK_GROUP_ID, moduleName, DEFAULT_EXTENSION, DEFAULT_CLASSIFIER, DEFAULT_VERSION));
```

Comment:
this is only affecting OOTB `stream` and `task` registrations.

Newly introduced libraries would typically be hand-registered, for now.